### PR TITLE
README: Add list of all charts required for full install

### PR DESCRIPTION
**Pull request by r4vi**
This commit lists the helm charts required for a full install rather than a
minimal install of the cluster

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,37 @@ helm install charts/$chart_name -f ...
...
+- `kuberos` (official `stable/kuberos`)
+- [logstash-auth0](./charts/logstash)
+- `metrics-server` (official `stable/metrics-server`)
+- [network-policies](./charts/network-policies)
+- [nfs-backup](./charts/nfs-backup)
+- [org-pipeline-moj-analytical-services](./charts/concourse-org-pipeline)
+- [pod-cleaner](./charts/pod-cleaner)
+- [priority-classes](./charts/priority-classes)
+- `prometheus-operator` (official `stable/prometheus-operator`)
+- [promethues-resources](./charts/prometheus-resources)
```
Comment (r4vi) [inline README.md]: @djsd123 I don't know about what's actually required for prometheus - do we still need the `cluster-metrics` / `stable/prometheus` chart?

**Comment by r4vi** [comment]
https://trello.com/c/qxHPBeFr/429-write-down-a-full-list-of-helm-charts-for-a-full-install

File: README.md (hunk 1 above)
Comment (djsd123) [reply README.md]: No, it's not needed in favour of the operator `kubectl get po --namespace monitoring -lapp=kube-state-metrics`

File: README.md [hunk 2]
```diff
@@ -26,9 +26,37 @@ helm install charts/$chart_name -f ...
...
 * `kube2iam` (official `stable/kube2iam` helm chart)
 * `rbac`
 * `cpanel`
-* `cpfrontend`
 * `aws-login`
+
+# Full set-up
+All of the above plus:
+- [airflow](./charts/airflow-k8s)
+- `cert-manager` (official `stable/cert-manager` helm chart)
```
Comment (djsd123) [inline README.md]: `- `cert-manager` (official `charts.jetstack.io/cert-manager` helm chart)`

**Pull request merged**